src/supermarket/app/lib/supermarket/fips.rb

```diff
@@ -2,7 +2,7 @@
 require "openssl"
 
 module Supermarket
-  class FIPS
+  class Fips
```

Comment:
The new autoloader zeitwerk imposes some strict naming conventions. Hence this change.